array-methods.js

```diff
@@ -65,7 +91,30 @@ var sumOfBankBalances = null;
   take each `amount` and add 18.9% interest to it rounded to the nearest cent
   and then sum it all up into one value saved to `sumOfInterests`
  */
-var sumOfInterests = null;
+
+let statesArray = ['WI', 'IL', 'WY', 'OH', 'GA', 'DE'];
+var sumOfInterests = 0;
+let filteredArray = [];
+
+filteredArray = bankBalances.filter(curr => {
```

Comment:
There's an easier way to filter by state using the statesArray and one of the other array methods in the docs. (not one of the functional array methods)